src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Consider using shorthand properties for padding and margin where possible to improve code conciseness. For example, `margin: 0` is already defined in the `body` selector, so the `margin: 0` in the universal selector `*` might be redundant unless you specifically want to reset margins for all elements.